testing/integration/broker/broker_test.go

```diff
@@ -1,49 +1,215 @@
 package broker_test
 
 import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"time"
+
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+
+	"os"
+
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	cfn "github.com/aws/aws-sdk-go/service/cloudformation"
+	awssqs "github.com/aws/aws-sdk-go/service/sqs"
+
+	"code.cloudfoundry.org/lager"
+	"github.com/alphagov/paas-service-broker-base/broker"
+	brokertesting "github.com/alphagov/paas-service-broker-base/testing"
+	"github.com/alphagov/paas-sqs-broker/sqs"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/pivotal-cf/brokerapi"
+	"github.com/pivotal-cf/brokerapi/domain"
+	uuid "github.com/satori/go.uuid"
 )
 
 const (
 	ASYNC_ALLOWED = true
 )
 
-type BindingResponse struct {
-	Credentials map[string]interface{} `json:"credentials"`
-}
+var (
+	PollingTimeout = time.Minute * 10
+)
 
 var _ = Describe("Broker", func() {
+	var (
+		brokerTester brokertesting.BrokerTester
+	)
 
-	BeforeEach(func() {
-		// instanceID = uuid.NewV4().String()
-	})
+	var (
```

Comment:
single var block?

Reply:
actually I did that intentionally ... trying to make it clearer what the system under test is by breaking it out on it's own ... since it was kind of lost in the `initialize()`